### ISSUE TITLE
zowe files upload dtu with .zosattributes doesn't upload files starting with '.'

### ISSUE THREAD

**Issue by dkelosky** [zowe/zowe-cli#767]
if you create a project with files that start with a `.` - like `.gitignore`.  This file does not appear to be uploaded even when excluded specified in a `.zosattributes` file:
```
# pattern local-encoding remote-encoding
node_modules -
# .gitignore -
# package-lock* -
```

Uploading via:
`zowe files upload dtu './/' '//a//ibmuser//native//node//hello2-wto' --attributes=.zosattributes`

**Comment by awharn** [comment]
This was recently fixed with the --include-hidden flag on the zowe files upload dtu command, and will be released in Zowe 2.4, or on NPM after the Zowe 2.3 release.

**Comment by adam-wolfe** [comment]
Hidden files may be included using the --include-hidden flag.
